extensions/amp-story-interactive/0.1/amp-story-interactive.css

```diff
@@ -110,3 +110,83 @@
     opacity: .4;
   }
 }
+
+/* Disclaimer CSS rules */
+.i-amphtml-story-interactive-disclaimer {
+  position: absolute !important;
+  color: black !important;
+  bottom: calc(100% - 1.75em) !important;
```

Comment:
It looks like this breaks when the quiz is at the top of the page:
<img width="492" alt="Screen Shot 2021-01-05 at 12 56 55 PM" src="https://user-images.githubusercontent.com/3860311/103681811-f53a8680-4f55-11eb-9c37-6bc775e62942.png">

Reply:
Good catch! I'd say we should check if the top position of the bubble leaves enough space for the bubble to expand up, and if not, move it down instead.

Filed a bug at #31799 since it would be a lot to include that in this PR